src/app/app.module.ts

```diff
@@ -8,9 +8,24 @@ import { AnalyticsModule, HotJarModule } from "hee-shared";
 import { AppRoutingModule } from "./app-routing.module";
 import { AppComponent } from "./app.component";
 import { SharedModule } from "./shared/shared.module";
+import { MainNavComponent } from "./main-nav/main-nav.component";
+import { MatMainNavComponent } from "./mat-main-nav/mat-main-nav.component";
+
+import { HttpClientModule } from "@angular/common/http";
+import { StatusBarComponent } from "./mat-main-nav/status-bar/status-bar.component";
+import { MobileMenuComponent } from "./mat-main-nav/mobile-menu/mobile-menu.component";
+import { DesktopMenuComponent } from "./mat-main-nav/desktop-menu/desktop-menu.component";
+import { MaterialModule } from "./shared/material/material.module";
 
 @NgModule({
-  declarations: [AppComponent],
+  declarations: [
+    AppComponent,
+    MainNavComponent,
+    MatMainNavComponent,
```

Comment:
it would be nice if `MainNavComponent, MatMainNavComponent, StatusBarComponent, MobileMenuComponent, DeskktopMenuComponent' could be wrapped by a `HeaderModule` which could be imported by the existing `SharedModule`

Reply:
True, ideally routing strategy decisions make these clear. E.g. empty app component with router-outlet. All legal notices, etc footer type of links and pages that dont care about auth status. error module with potential error routes and components, navigation seems like it will need auth and roles but not just yet. I was going to put these into hee-shared and export from there initially but will put into header as suggested for now.

Reply:
created module for related components. done

Reply:
sweet